main.go

```diff
@@ -65,10 +60,20 @@ func main() {
 				} else {
 					connectionString = parsedPort
 				}
-				r := initServer()
+
+				// Make sure to handle ~/ in dbpath
+				var dbPath string
+				if c.String("dbpath") != "" {
+					dbPath = fmt.Sprintf("%q", c.String("dbpath"))
+				}
+				db := job.GetBoltDB(dbPath)
+
+				// Create cache
+				cache := job.NewMemoryJobCache(db, 5*time.Second)
```

Comment:
@ajvb the 5 seconds definitely shouldn't be hard coded. Throw it in a constants for now, then perhaps a cmd flag later.